infrastructure/rdfdb/cdk/docker/scripts/setup.sh

```diff
@@ -4,14 +4,17 @@ set -e
 echo 'org.apache.tomcat.util.digester.PROPERTY_SOURCE=org.apache.tomcat.util.digester.EnvironmentPropertySource' >> /usr/local/tomcat/conf/catalina.properties
 echo "RDF4J_DATA_DIR is set to: ${RDF4J_DATA_DIR}"
 
-/usr/local/tomcat/setup_web_auth.sh
+# /usr/local/tomcat/setup_web_auth.sh
 
 echo "showing rdf4j data directory"
 ls -l /rdf4j-data
 
-export JAVA_OPTS="-Dorg.eclipse.rdf4j.appdata.basedir=${RDF4J_DATA_DIR}"
-export CATALINA_OPTS="-Dorg.eclipse.rdf4j.appdata.basedir=${RDF4J_DATA_DIR}"
-echo "JAVA_OPTS=\"\$JAVA_OPTS -Dorg.eclipse.rdf4j.appdata.basedir=${RDF4J_DATA_DIR}\"" >> /usr/local/tomcat/bin/setenv.sh
+# Set Java options
+JAVA_OPTS="-Dorg.eclipse.rdf4j.appdata.basedir=${RDF4J_DATA_DIR} -Xmx3g -Xms1g -XX:MaxMetaspaceSize=512m -XX:+UseG1GC -XX:MaxGCPauseMillis=200"
+export JAVA_OPTS
```

Comment:
Can't this just be 
`export JAVA_OPTS="-Dorg.eclipse.rdf4j.appdata.basedir=${RDF4J_DATA_DIR} -Xmx3g -Xms1g -XX:MaxMetaspaceSize=512m -XX:+UseG1GC -XX:MaxGCPauseMillis=200"` ?

Reply:
Yes, changing.